src/test/java/teammates/test/cases/browsertests/InstructorFeedbackEditPageUiTest.java

```diff
@@ -876,7 +876,22 @@ private void testDeleteQuestionAction(int qnNumber) throws MaximumRetriesExceede
         feedbackEditPage.waitForConfirmationModalAndClickOk();
         feedbackEditPage.verifyStatus(Const.StatusMessages.FEEDBACK_QUESTION_DELETED);
         assertNull(getFeedbackQuestion(courseId, feedbackSessionName, qnNumber));
+    }
+
+    private void testEditWithEmptyQuestionTextThenDeleteQuestionAction(int qnNumber)
+            throws MaximumRetriesExceededException {
+        ______TS("qn " + qnNumber + " edit the question with invalid input then delete question");
+
+        feedbackEditPage.clickEditQuestionButton(qnNumber);
+        feedbackEditPage.fillQuestionTextBox("", qnNumber);
+        feedbackEditPage.clickSaveExistingQuestionButton(qnNumber);
+        String expectedString = "Please enter a valid question. The question text cannot be empty.";
+        feedbackEditPage.verifyStatus(expectedString);
```

Comment:
Use `Const#FEEDBACK_QUESTION_TEXTINVALID`?  And can you rename it to use underscores?

Optionally, you can also move that constant to the appropriate test page because it is only used in tests.